src/sections/Meshery/Meshery-platforms/index.js

```diff
@@ -75,36 +78,22 @@ const supported_platforms = [
       <>
         <h2>Helm Chart</h2>
         <p>Install on Kubernetes using Helm:</p>
-        <Code codeString={dedent`kubectl create namespace meshery
-      helm repo add meshery https://meshery.io/charts/
-      helm install meshery meshery/meshery -n meshery`} />
-      </>
-    )
-  },
-  {
-    icon: HomeBrew,
-    name: "HomeBrew",
-    steps: (
-      <>
-        <h3>Brew User</h3>
-        <p>Install on Mac or Linux using Homebrew:</p>
-        <Code codeString={dedent`brew tap layer5io/tap
-                    brew install mesheryctl
-                    mesheryctl system start`}
-        />
+        <Code codeString={dedent`helm repo add meshery https://meshery.io/charts/
+        helm install my-meshery meshery/meshery --version 2.1.2`
+        } />
       </>
     )
   },
+  
   {
     icon: Kind,
-    name: "Kind",
+    name: "KinD",
```

Comment:
@debjani1413 Isn't this supposed to be Kind or kind itself?  I was referring to this, https://kind.sigs.k8s.io/

Reply:
Okay sure, sorry for the mistake!

Reply:
No Probs, Lmk once you have pushed the changes, will re-review it once more :)